docs/pages/reference/session-recording.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Teleport Session Recording
+description: An overview of Teleport's session recording and its configuration
+---
+
+This page provides an overview of how Teleport records sessions and the various
+configuration options that apply to session recordings.
+
+Teleport supports two different recording configurations: synchronous recording
+and asynchronous recording. Additionally, Teleport administrators can configure
+where the recording takes place: at the node or at the proxy.
+
+This results in 4 possible sesssion recording configurations:
+
+- `node-sync`: synchronous recording performed by the SSH node
+- `node`: asynchronous recording performed by the SSH node
+- `proxy-sync`: synchronous recording performed at the proxy
+- `proxy`: asynchronous recording performed at the proxy
+
+This is a cluster-wide configuration option and applies to the entire
+Teleport cluster. If you need to apply different recording configuration to
+different sets of resources, we recommend setting up Trusted Clusters with
+their own recording configurations.
+
+<Admonition type="note" title="Desktop Access">
+Desktop Sessions are always recorded at Teleport's Windows Desktop Service
+since there is no Teleport software running on the "node" (desktop).
+
+For this reason, Teleport Desktop Access treats `node` and `proxy` identically
+(perform synchronous desktop recording) and `node-sync` and `proxy-sync` identically
+(perform asynchronous desktop recording).
```

Comment:
Suggested to address the discussion thread above.

```suggestion
Desktop, Database, and Kubernetes Sessions are always recorded at the host running
the Teleport service for that session type, since there is no Teleport software
running on the "node" (desktop, database, or Kubernetes cluster).

For this reason, Teleport Desktop, Database, and Kubernetes Access treats `node`
and `proxy` identically (perform synchronous recording) and `node-sync` and
`proxy-sync` identically (perform asynchronous recording).
```

Reply:
Adjusted after discussion.